test/run_test.py

```diff
@@ -324,6 +324,7 @@ def skip_test_p(name: str) -> bool:
 CI_SERIAL_LIST = [
     "test_nn",
     "test_fake_tensor",
+    "test_set_sequence_nr",
```

Comment:
Why does this need to be in the serial list? It should be ok to run it anytime?

Reply:
oh whoops I mostly had no idea how to write a test and just templated off of test_fake_tensor